### PR TITLE
Added Headphones

### PR DESCRIPTION
**Pull request by bob28**
Added four headphone icons to the files

### CODE REVIEW

**Comment by quozl** [comment]
Thanks.  I'd like to review, but can't see the icons in context.  Have you a design screenshot or feature page?

File: icons/scalable/device/headphone-full.svg [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
...AQEaAgIwAARkBAjICBGQECMgIEJARICAjQEBGgICMAAEZAQIyAgRkBAjICBCQESAgI0BARoCAjAABGQECMgIEZAQIyAgQkBEgICNAQEaAgIwAARkBAjICBGQECMgIEJARICAjQEBGgICMAAEZAQIyAgRkBAjICBCQESAgI0BARoCAjAABGQECMgIEZAQIyAgQkBEgICNAQEaAgIwAARkBAjICBGQECMgIEJARICAjQEBGgICMAAEZAQIyAgRkBAjICBCQESAgI0BARoCAjAABGQECMgIEZAQIyAgQkBEgICNAQEaAgIwAARkBAjICBGQECMgIEJARICAjQEBGgICMAAEZAQIyAgRkBAjICBCQESAgI0BARoCAjAABGQECMgIEZAQIyAgQkBEgICNAQEaAgIwAARkBAjICBGQECMgIEJARICAjQEBGgICMAAEZAQIyAgRkBAjICBCQESAgI0BARoCAjAABGQECMgIEZAQIyAgQkBEgICNAQEaAgIwAARkBAjICBGQECMgIEJARICAjQEBGgICMAAEZAQIyAgRkBAjICBCQESAgI0BARoCAjAABGQECMgIEZAQIyAgQkBEgICNAQEaAgIwAARkBAjICBGQECMgIEJARICAjQEBGgICMAAEZAQIyAgRkBAjICBCQESAgI0BARoCAjAABGQECMgIEZAQIyAgQkBEgICNAQEaAgIwAARkBAjICBGQECMgIEJARICAjQEBGgICMAAEZAQIyAgRkBAjICBCQESAgI0BARoCAjAABGQECMgIEZAQIyAgQkBEgICNAQEaAgIwAARkBAjICBGQECMgIEJARICAjQEBGgICMAAEZAQIyAgRkBAjICBCQESAgI0BARoCAjAABGQECMgIEZAQIyAgQkBEgICNAQEaAgIwAARkBAjICBGQECMgIEJARICAjQEBGgICMAAEZAQIyAgRkBAjICBCQESAgI0BARoCAjAABGQECMgIEZAQIyAgQkBEgICNAQEaAgIwAARkBAjICBGQECMgIEJARICAjQEBGgICMAAEZAQIyAgRkBAjICBCQESAgI0BARoCAjAABGQECMgIEZAQIyAgQkBEgICNAQEaAgIwAARkBAjICBGQECMgIEJARICAjQEBGgICMAAGpbdZae38/qX5DGQJJe9EAAAAASUVORK5CYII=" y="-0.53994524"/>
```
Comment (samdroid-apps) [inline icons/scalable/device/headphone-full.svg]: The icon needs to be fully svg.  That means that it should include no images inside of it.

Sugar changes the stroke and fill colours of the icon based on the context, but that only works if everything is svg.

**Comment by samdroid-apps** [comment]
Also, maybe instead of low & muted, do like 020 (20%) 040 (40%), etc.  This is how the speaker icons are organised, and what you should try and do too.

**Comment by samdroid-apps** [comment]
@quozl it is just gci task to make the headphone icons.  This is a pre-requisite to displaying them in the frame depending on if the headphones are plugged in.

**Comment by bob28** [comment]
Hello, this headphone task is a GCI task and I just had go create a set of icons when the headphones are plugged in.  I hope you don't mind.  Thank you

**Comment by quozl** [comment]
In reviewing technical contributions, I don't treat GCI tasks with any more or less exactness.

I don't think we need to host icons in the repository that aren't used yet, so I'd like to see the use of them first.

I agree the icon needs to be compatible with the Sugar stroke and fill changes, that are made to colour icons according to user preference.

**Pull request closed without merging**